pkgs/top-level/all-packages.nix

```diff
@@ -15945,7 +15919,7 @@ with pkgs;
 
   hugs = callPackage ../development/interpreters/hugs { };
 
-  inherit (javaPackages) openjfx11 openjfx15 openjfx17 openjfx19 openjfx20 openjfx21 openjfx22;
+  inherit (javaPackages) openjfx11 openjfx17 openjfx19 openjfx20 openjfx21 openjfx22;
```

Comment:
And `openjfx11` is still alive?

Reply:
OpenJDK 11 is LTS, so OpenJFX 11 stays with it.